components/calendar/src/cal/chinese.rs

```diff
@@ -158,6 +169,12 @@ pub trait Rules: Clone + core::fmt::Debug + crate::cal::scaffold::UnstableSealed
 /// required before 1900, clients can implement their own [`Rules`] type
 /// using data such as from the excellent compilation by [Yuk Tung Liu].
 ///
+/// The precise behavior of this calendar may change in the future if:
+/// - New ground truth in China is established, either through published government sources or most almanacs
```

Comment:
I don't think this should ever adopt data from "most almanacs". this is GBT

```suggestion
/// - New ground truth is established by published government sources
```